LSE-61.tex

```diff
@@ -2389,6 +2522,16 @@ \subsubsection{Keep Historical Alert Archive}
 OSS-REQ-0128:
 Alerts \newline
 
+\subsubsection{Archiving Camera Test Data}
+
+\label{DMS-REQ-0372}
+\textbf{ID:} DMS-REQ-0372
+
+\textbf{Specification:} The DMS shall be able to archive a designated subset of Camera test data and make it available in an environment matching the data backbone interfaces.
```

Comment:
I don't think the term "data backbone" should be introduced to the OSS at this late stage.

Isn't what's really meant, at OSS level, "matching that provided for the LSST Data Products" as defined in OSS-REQ-0176 and OSS-REQ-0177?

Reply:
It was pointed out subsequently that this is not a change, but an artifact of moving this requirement to a different location in the document, so the text of this requirement is not open for editing.

Reply:
(It was also pointed out that I incorrectly wrote "OSS" in my comment.  Brain bubble.)